helpers/httpApi.js

```diff
@@ -176,6 +176,24 @@ var middleware = {
 		} else {
 			next();
 		}
+	},
+	flushCache: function (logger, cache, req, res, next) {
```

Comment:
Express calls this function internally with `(req, res, next)` parameters. So, (based on my knowledge), we need to specify `req` parameter here.